AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/MySqlSettingRepository.cs

```diff
@@ -11,6 +11,7 @@ public class MySqlSettingRepository : ISettingRepository
     // Creates SettingsContext object
     private readonly SettingsContext _settingsContext;
     private readonly IBidirectionalConverter<Model.Setting, Setting> _settingsConverter;
+    private readonly IBidirectionalConverter<Model.SettingGroup, Model.SettingGroup> _settingsConverter;
```

Comment:
```suggestion
    private readonly IBidirectionalConverter<Model.SettingGroup, SettingGroup> _settingGroupConverter;
```

Reply:
Gotta give this a different name and make sure the types specified in the "<>"
match the implementation https://github.com/WSU-4110/AppSettingsManager/blob/main/AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/Converters/SettingGroupBiverter.cs